ui/job-view/App.jsx

```diff
@@ -217,6 +223,25 @@ class App extends React.Component {
     return repo;
   }
 
+  async setLandoRevision() {
+    const { pushRoute } = this.props;
+    const params = getAllUrlParams();
+    const landoCommitID = params.get('landoCommitID');
+    let revision = params.get('revision');
+
+    if ((!revision || revision === 'undefined') && landoCommitID) {
+      const { data } = await getData(getLandoJobsUrl(landoCommitID));
+
+      revision = data.commit_id;
```

Comment:
At the moment, a request directly after running `./mach try` will return an empty value. This is likely a common use case for developers.

1. There should be repeated attempts to get the revision, e.g. with `setInterval` and a two minutes interval.
2. If the revision is not available yet, a list with most recent pushes is loaded. For this case, a 'Waiting for push with revision' page should be shown ([example](https://treeherder.mozilla.org/jobs?repo=try&revision=2955db5debe35fb731fb9d27392bf81001e18961)).
3. The `landoCommitID` parameter should be dropped when the revision hash is found.

Reply:
I added a setInterval call to repeat the attempt, and I've also added a custom PushLoadErrors section for the lando commit IDs.

Note that I had to modify some of the methods in `pushes.js` to properly handle the lack of a revision from the lando API call.